getspice.py

```diff
@@ -208,9 +216,15 @@ def _download(self):
             import urllib3
 
             try:
-                # Create a PoolManager
-                https = urllib3.PoolManager(cert_reqs='CERT_REQUIRED',
-                                            ca_certs=certifi.where())
+                if proxy:
+                    # Create a ProxyManager
+                    https = urllib3.ProxyManager(proxy,
```

Comment:
I think you could just have proxy be a None or a Dict, and use the proxymanager only instead of having the if statement / the else and such

Reply:
I agree but it looks like neither `ProxyManager` nor `PoolManager` are able to manage `dict` or `None`. Unfortunately the if/else block seems to be require 😞. I also added `https` vs. `http` checks based on the previous `proxies` parsing from env variables (with `https` first).